app/main.py

```diff
@@ -1 +1,31 @@
-# write your code here
+class Animal:
+    alive = []
+
+    def __init__(self, name: str, health: int = 100, hidden: bool = False)\
+            -> None:
```

Comment:
don't use "\", better do something like this:
    def __init__(self, name: str, health: int = 100, 
                      hidden: bool = False) -> None:

Reply:
Done.

Reply:
Good job